Adding enigma2 docs

fbradyirl opened the pull request:
**Description:**
Adding enigma2 docs. (this is based on comments from a stale review here: #5984 ).

**Pull request in [home-assistant](https://github.com/home-assistant/home-assistant) (if applicable):** home-assistant/home-assistant#21271

## Checklist:

- [x] Branch: Fixes, changes and adjustments should be created against `current`. New documentation for platforms/components and features should go to `next`.
- [x] The documentation follow the [standards][standards].

[standards]: https://home-assistant.io/developers/documentation/standards/

Commit:
Adding enigma2 docs

Commit:
Adding enigma2 docs

Commit:
Adding enigma2 docs

Commit:
Adding enigma2 docs

frenck committed:
:pencil2: Language tweaks

frenck committed:
:pencil2: Language tweaks

fbradyirl committed:
Remove optional config

fbradyirl committed:
Remove optional config

fbradyirl committed:
Update media_player.enigma2.markdown

fbradyirl committed:
Update media_player.enigma2.markdown

klaasnicolaas committed:
🚜 Merges/Redirect Rachio component pages

klaasnicolaas committed:
:pencil2: Tweak

theaquarium committed:
Auth providers doc should use gender neutral language. (#8660)

klaasnicolaas committed:
🚜 Merges/Redirect Rainmachine component pages (#8659)

klaasnicolaas committed:
🚜 Merges/Redirect Homeworks component pages (#8658)

klaasnicolaas committed:
🚜 Merges/Redirect Unifi component pages (#8657)

klaasnicolaas committed:
🚜 Merges/Redirect Sense component pages (#8655)

fabaff committed:
🚜 Merges/Redirect Rachio component pages (#8656)

* 🚜 Merges/Redirect Rachio component pages

* :pencil2: Tweak

NateRobinsonS committed:
Update googlehome.markdown (#8663)

Corrected device entity id format (inserted missing underscore) in line 74.

Bhaaf committed:
Added Denon AVR-X1500H (#8664)

Bought and tested Denon AVR-X1500H

allanpersson committed:
Update sensor.rejseplanen.markdown (#8667)

You need to change the "stop_name" to a location where you want id for, else you only get a short random list of id's :)

Commit:
Update release and date

Commit:
Update release and date

Commit:
Merge branch 'next' of github.com:home-assistant/home-assistant.io into next

klaasnicolaas commented on source/_components/media_player.enigma2.markdown:
```diff
@@ -0,0 +1,35 @@
+# Example configuration.yaml entry
+media_player:
+  - platform: enigma2
+    host: 192.168.1.12
```
```suggestion
    host: IP_ADDRESS
```

klaasnicolaas committed:
Update source/_components/media_player.enigma2.markdown

Co-Authored-By: fbradyirl <fbradyirl@users.noreply.github.com>

klaasnicolaas committed:
Update source/_components/media_player.enigma2.markdown

Co-Authored-By: fbradyirl <fbradyirl@users.noreply.github.com>

jeffh0821 committed:
Clarify use of alternate ports in MariaDB connections (#8670)

partytimeexcellent committed:
Fix example automation not triggering (#8673)

I couldn't get the "ios.notification_action_fired" event to trigger an automation using this guide. Through trial and error, I found the automation yaml needs an additional line "data:" below "event_data:"

brianjking committed:
person component standardization & clarification for how to use configuration panel. (#8671)

* Capitalize OwnTracks to comply with HomeAssistant [documentation standards](https://developers.home-assistant.io/docs/en/documentation_standards.html)

* Clarify that if you're going to use the configuration panel for configuring the `person` component you need to add `person:` to your `configuration.yaml` file and restart Home Assistant

balloob committed:
Release 88.1

ludeeus committed:
Adds redirect_from /components/remote.harmony/

robbiet480 committed:
Merge pull request #8680 from home-assistant/ludeeus-patch-1

Adds redirect_from /components/remote.harmony/

pvizeli committed:
Update HassOS 2.10

jamieshaw committed:
Fix 'hours' key name for Magicseaweed (#8682)

Documentation incorrectly referenced 'hour' with was flagged as invalid starting in 0.88.

klaasnicolaas committed:
🚜 Merges/Redirect Melissa component pages

increddibelly committed:
added NU7090, removed duplicate (#8684)

klaasnicolaas committed:
🚜 Merges/Redirect Vultr component pages (#8683)

fabaff committed:
🚜 Merges/Redirect Melissa component pages (#8685)

klaasnicolaas committed:
🚜 Merges/Redirect IOTA component pages (#8686)

elupus committed:
Remove invalid configuration parameter (#8688)

Fix for https://github.com/home-assistant/home-assistant/issues/21300

elupus committed:
Remove invalid configuration parameter (#8688)

Fix for https://github.com/home-assistant/home-assistant/issues/21300

iantrich committed:
Update changelog.markdown (#8690)

iantrich committed:
image-element missing aspect_ratio config (#8676)

rytilahti committed:
Add documentation for TP-Link Smart Home devices (#7338)

* Update tplink documentation

* Make configuration linkable, add documentation for list elements

* :pencil2: Tweaks

* Add KL series of bulbs from #8134

* Add LB230 from #8090

* Add extracting energy sensor data from the switch platform

* fix indent

* add 0.89 for version

* re-add HS103

fabaff committed:
Merge branch 'current' into next

AlecRust committed:
Fix link (#8694)

itamarbareket committed:
Elaborate on Caddy configuration process (#8692)

* Elaborate on Caddy configuration process

* Changed NGINX => Caddy

timmo001 committed:
:ambulance: Fixes link to access tokens (#8695)

iantrich committed:
fix typo (#8696)

dshokouhi committed:
Allow user to disable device tracker (#8701)

Useful for users who only wish to track alarms or only track devices from specific google home units.

klaasnicolaas committed:
🚜 Merges/Redirect Doorbird component pages (#8702)

klaasnicolaas committed:
🚜 Merges/Redirect Danfoss_air component pages (#8699)

* 🚜 Merges/Redirect Danfoss_air component pages

* :pencil2: Tweak

klaasnicolaas committed:
🚜 Merges/Redirect LCN component pages (#8698)

klaasnicolaas committed:
🚜 Merges/Redirect Apcupsd component pages (#8697)

fabaff committed:
Merge branch 'current' into next

cstuder committed:
Adds documentation for Edimax power sensor (#8677)

* Edimax: Updates documentation link to https

* Edimax: Adds documentation for power consumption reporting

ludeeus committed:
Change customize to core (customize)

DubhAd committed:
Merge pull request #8707 from home-assistant/rewording-core-reload

Change customize to core (customize)

fabaff committed:
Add freedaily option (#8708)

philipperequile committed:
Added 2 new PLC types DINT and UDINT (#8700)

1) added 2 new PLC types. 
2) Correction in the configuration.yaml example: port is typically 801 or 851, not 48898 (48898 is the TCP port on the PLC. 801 (twincat 2) or 851 (twincat 3) are the default ADS ports under which the PLC runtime is accessible.

Limych committed:
Added OPNsense to table (#8711)

l-mb committed:
Clarify which password to use with velux component (#8706)

* Clarify which password to use with velux component

Previous examples everywhere use the "velux123" string. That is the web login password the KLF 200 defaults to. The password actually required is the other one, which is both significantly more complex and set elsewhere.
Changing the example will hopefully help others to not waste as much time on this as I did ;-)

* Minor change as stated in the guidelines

DubhAd committed:
Clarity edit (#8709)

Adding a few words, to make it clearer that a disabled automation doesn't run. Also removing the redundant, and no longer relevant, final paragraph.

Gamester17 committed:
Update zha.markdown to mention ZHA Device Handlers (#8681)

* Update zha.markdown to mention ZHA Device Handlers

Update zha.markdown to mention the ZHA Device Handlers (a.k.a. ZHA quirks handler) by @dmulcahey

This proposes adding text: "Zigbee devices that deviate or do not fully conform from standard specification set by the [Zigbee Alliance](https://www.zigbee.org) may require the development of custom [ZHA Device Handlers](https://github.com/dmulcahey/zha-device-handlers) (a.k.a. ZHA quirks handler implementation) to for all their functions to work properly with the ZHA component in Home Assistant."

* Minor changes

thibmaek committed:
Update config options for sensor.discogs (#8214)

* Update config options for sensor.discogs

* Update monitored_conditions

JeffLIrion committed:
Add ADB server functionality to the Fire TV component (#8641)

blindahl committed:
Update timer.markdown (#8719)

Adjusted heading level from H4 to H3 to match other tiles on same level.

olbjan committed:
Corrected example automation (#8726)

The trigger in the example automation contained the line 'data:' below the event_data, which is redundant.
If used this way the automation triggers on any iOS action and ignores the actionName.

Cooper-Dale committed:
solution for a bug in ?Motion|motionEye (#8725)

more info here: https://github.com/ccrisan/motioneye/issues/1062#issuecomment-466714428

alengwenus committed:
Add documentation for LCN cover platform (#8234)

* Add documentation for LCN cover platform

* Updated hass version number

* Removed old cover.lcn.markdown

* Added redirect_from

* Update lcn.markdown

klaasnicolaas committed:
🚜 Merges/Redirect Habitica component pages (#8728)

andrewsayre committed:
Update SmartThings switch platform docs (#8713)

klaasnicolaas committed:
🚜 Merges/Redirect SMHI component pages (#8731)

klaasnicolaas committed:
🚜 Merges/Redirect USPS component pages (#8732)

ToSa27 committed:
google maps allow multiple accounts (#8020)

cookie file extension to allow multiple accounts to be scanned

marcogazzola committed:
xiaomi vacuum with zone and reps (#8047)

* xiaomi vacuum with zone and reps

* xiaomi vacuum with zone and reps

* xiaomi vacuum with zone and reps

* xiaomi vacuum with zone and reps

* xiaomi vacuum with zone and reps

* add reps template

* rytilahti tips

* rytilahti tips

* automation editor fix

* Revert "automation editor fix"

This reverts commit 58bb3ad6a77c90b2fe4b080c1e36a7db18897224.

* rytilahti tips

* data_template schema

* conflict resolve

* conflict

* fix merge error

* :pencil2: Tweak

mzdrale committed:
Added documentation for Ubee router device tracker (#7955)

* Added Documentation for Ubee Router Device Tracker.

* Added Documentation for Ubee Router Device Tracker (this time for real)

* Updated Ubee logo to be visible on white background and changed title

* Changed HA version to 0.87.

* Changed HA version to 0.88.

* Bumped to 0.89 version

* :pencil2: Tweaks

RomRider committed:
Neato supports custom cleaning (#8421)

* Neato supports custom cleaning

* Adding some details about the new service

* More details on the new service

* Revert back the domain to vacuum as requested

msvinth committed:
Health_idx (#8613)

alistairg committed:
Add documentation for sony_projector switch component (#8390)

* Add documentation for sony_projector switch component

* :arrow_up: ha_release 0.89

lapy committed:
Update traccar documentation with new parameters (#8618)

* Update traccar documentation with new parameters 

Added scan_interval and monitored_conditions

* Add clarification on monitored_conditions

* :pencil2: Removes scan_interval

UnrealKazu committed:
Add Docker specific explanation for PS4 component (#8629)

The new PS4 component requires some extra settings from Home Assistant if one is using Docker. I've added an entry in the docs to make sure these settings are clear for those who use Docker.

andrewsayre committed:
Add SmartThings cover platform and update cover device classes (#8625)

* Add SmartThings Cover docs

* Update cover component device classes

* Add additional cover types

* Sync device class descriptions

* :pencil2: Tweaks

scstraus committed:
Add an example with a template (#8712)

* Add an example with a template

I found it very hard to find an example template for IMAP email sensor, so I think it would be helpful to include a working one here. This one works correctly and has been tested extensively with a popular cloud service.

* Update sensor.imap_email_content.markdown

* :pencil2: Tweaks

JBassett committed:
utility_meter net metering support (#8631)

* utility_meter net metering support

Add documentation around net metering for utility_sensor.

* Change config item to new name

New name to make it more obvious what the configuration does.

* :pencil2: Tweaks

* :pencil2: Tweak

allanpersson committed:
Update sensor.rejseplanen.markdown (#8730)

* Update sensor.rejseplanen.markdown

Added important description on how to use the "destination" filter.

* :pencil2: Tweaks

amelchio committed:
Sonos updates (#8724)

klaasnicolaas committed:
🚜 Merges/Redirect OctoPrint component pages (#8744)

klaasnicolaas committed:
🚜 Merges/Redirect Point component pages (#8743)

* 🚜 Merges/Redirect Point component pages

* :pencil2: Tweak

klaasnicolaas committed:
🚜 Merges/Redirect UpCloud component pages (#8742)

klaasnicolaas committed:
🚜 Merges/Redirect Netgear LTE component pages (#8740)

klaasnicolaas committed:
🚜 Merges/Redirect WirelessTag component pages (#8739)

klaasnicolaas committed:
🚜 Merges/Redirect Mochad component pages (#8738)

klaasnicolaas committed:
🚜 Merges/Redirect NuHeat component pages (#8737)

klaasnicolaas committed:
🚜 Merges/Redirect Locative component pages (#8735)

rohankapoorcom committed:
Remove the google travel time update service (#8749)

klaasnicolaas committed:
🚜 Merges/Redirect Fibaro component pages (#8754)

glogiotatidis committed:
Add timeout option docs for sensor.rest and binary_sensor.rest. (#8161)

frenck committed:
:books: Updates Toon documentation to match rewrite (#8627)

erikbent committed:
Only the host name is used, not the protocol (#8756)

I searched a long time why there was nothing stored in graphite. I had to remove the protocol.

thibmaek committed:
Update new configuration options for sensor.nmbs (#8201)

* Add docs

* Update to use the standars

* Update source/_components/sensor.nmbs.markdown

Co-Authored-By: thibmaek <thibault.maekelbergh@iCloud.com>

* Update source/_components/sensor.nmbs.markdown

Co-Authored-By: thibmaek <thibault.maekelbergh@iCloud.com>

* Update source/_components/sensor.nmbs.markdown

Co-Authored-By: thibmaek <thibault.maekelbergh@iCloud.com>

* Update source/_components/sensor.nmbs.markdown

Co-Authored-By: thibmaek <thibault.maekelbergh@iCloud.com>

* Update source/_components/sensor.nmbs.markdown

Co-Authored-By: thibmaek <thibault.maekelbergh@iCloud.com>

* Update source/_components/sensor.nmbs.markdown

Co-Authored-By: thibmaek <thibault.maekelbergh@iCloud.com>

* Update source/_components/sensor.nmbs.markdown

Co-Authored-By: thibmaek <thibault.maekelbergh@iCloud.com>

* Update source/_components/sensor.nmbs.markdown

Co-Authored-By: thibmaek <thibault.maekelbergh@iCloud.com>

* Add nmbs logo

* Add note on where to find station identifiers

* Update source/_components/sensor.nmbs.markdown

Co-Authored-By: thibmaek <thibault.maekelbergh@iCloud.com>

* Change release version

* Correct key

* Update with new conf options for sensor.nmbs

* :pencil2: Tweaks

* :ambulance: Reverts ha_version change

rohankapoorcom committed:
Update docs for the new iperf3 component (#8750)

andrewsayre committed:
Add SmartThings Scene platform docs (#8718)

* Add SmartThings Scene docs

* :pencil2: Tweak

awarecan committed:
Add config for trusted networks auth provider (#8605)

* Update providers.markdown

* Update providers.markdown

ljmerza committed:
Reddit Sensor (#8675)

* automated commit 21/02/2019 20:05:00

* :pencil2: Tweaks

frenck committed:
Merge branch 'current' into next

rosscullen committed:
API link updated and charge for API (#8741)

Update to API Link (previous link invalid. Added details of cost introduced to use their cloud API. I was informed of this when I recently requested my client_secret.

I was given the following from Smappee support but has yet to configure with local API via MQTT (Mosquitto & Node Red). If anyone knows how best to write this information into the page, would appreciate it.

Here are the links they provided:
How to find the UUID.pdf
https://support.smappee.com/attachments/token/y7MJ5O6e9P2kzuniuNfP5glFw/?name=How+to+find+the+UUID.pdf

Smappee local MQTT topics -20-09-2018.pdf
https://support.smappee.com/attachments/token/o5gKlg0jqbK2mVh0B6yfHArCy/?name=Smappee+local+MQTT+topics+-20-09-2018.pdf

Node Red, Smappee nodes:
https://flows.nodered.org/node/@smappee/node-red-contrib-smappee

Petro31 committed:
Add person domain to Waze Travel Time (#8757)

Linked PR

https://github.com/home-assistant/home-assistant/pull/21471

hybridparadigm committed:
Updates for python36 and scl (#8721)

* Updates for python36 and scl

Latest changes to installing the scl environment don't include the scl command, scl-utils is needed along w/ the proper repository in Centos7 to be enabled via RH and then downloaded to the environment which you are building and then SCL commands will work and setting python environment for bash will work too.

* Minor changes

kevintuhumury committed:
Add cpu_use_percent as a resource to the Glances sensor. (#8752)

SNoof85 committed:
Freebox added informations for initial setup (#8691)

* Added informations for initial setup

* :pencil2: Tweak

ant-car committed:
Added aux and panic services description (#8758)

The Aux commands work on my new D16XD.  I haven't tested panic.

ctborg committed:
Adds docs for enabling I2C on Hass.io  (#8144)

* Create enable_i2c.markdown

* Update sensor.bme680.markdown

* Update hassio_navigation.html

* :pencil2: Tweaks

* :pencil2: Tweaks

jurgenhaas committed:
Add command line argument --init (#8232)

See also https://github.com/home-assistant/home-assistant/issues/20260, the --init argument makes sure that the python service does not run with PID 1 which on some host systems can not be killed, i.e. restarting the service from within the UI is unreliable without that argument.

davidjb committed:
Clarify that http.base_url doesn't support paths (#8615)

Explains that paths aren't supported in base_url so someone attempting to set settings doesn't expect something that isn't currently possible.  Related: https://github.com/home-assistant/home-assistant/issues/21113

fbradyirl commented:
@klaasnicolaas are you happy with this PR?

klaasnicolaas commented:
I'm missing the `name` and `port` variable 😉

balloob committed:
Release 0.88.2

balloob committed:
Merge remote-tracking branch 'origin/current' into next

iantrich committed:
Update changelog.markdown (#8771)

gorynychzmey committed:
Added description for source priority. (#8773)

Commit:
Update to add port and name

Commit:
Merge branch 'next' of github.com:fbradyirl/home-assistant.io into next
